reactDashboard/src/components/Facts/Facts.css

```diff
@@ -25,19 +29,22 @@
   background-color: #ffffff;
 }
 
+.darkmode .facts-component {
+  background-color: rgb(56, 56, 56);
+  box-shadow: none;
+}
+
 .facts-icon {
   padding: 5px;
-  background-color: #ffffff;
   height: 5rem;
-  width: 5rem;
   margin: 0;
 }
 
 .facts-guidance {
   margin: 0 5px 0 10px;
   padding-right: 5px;
   width: 100%;
-  background-color: #ffffff;
+  background-color: inherit;
```

Comment:
Is this necessary - thought the default for background colour was transparent? What happens if you just delete the line?